sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.net8.0.cs

```diff
@@ -79,7 +79,7 @@ public SearchClient(System.Uri endpoint, string indexName, Azure.Core.TokenCrede
     }
     public partial class SearchClientOptions : Azure.Core.ClientOptions
     {
-        public SearchClientOptions(Azure.Search.Documents.SearchClientOptions.ServiceVersion version = Azure.Search.Documents.SearchClientOptions.ServiceVersion.V2024_11_01_Preview) { }
+        public SearchClientOptions(Azure.Search.Documents.SearchClientOptions.ServiceVersion version = Azure.Search.Documents.SearchClientOptions.ServiceVersion.V2025_03_01_Preview) { }
```

Comment:
Forgot to update the API after we changed the service version in the last PR.

Reply:
I believe CI is not running because I'm merging to a feature branch. I'll move the work to main after this PR.